src/mlpack/methods/reinforcement_learning/async_learning.hpp

```diff
@@ -118,7 +118,7 @@ class AsyncLearning
   TrainingConfig config;
 
   //! Locally-stored global learning network.
-  NetworkType learningNetwork;
+  NetworkType& learningNetwork;
```

Comment:
Don't think the reference is needed here. Maybe I'm missing something.

Reply:
I think if I remove the reference from here, the `network` reference passed by the user to the constructor will not be updated. Let me know what do you think

Reply:
We can't easily serialize reference types, so probably this needs to stay what it was before.  Basically a reference has to be set at creation time and can never be changed, so if a class has a reference we have to make a special constructor for deserialization... it's not pretty and if we can avoid it I might recommend that. :)